### ISSUE TITLE
The code will stop moving when building the model

### ISSUE THREAD

**Issue by panicwq** [jesse1029/Fake-Face-Images-Detection-Tensorflow#2]
The specific display is as follows：

GO!!
Preparing the training & validation data...
Found 1279 training images, and 387 validation images...
Preparing the training model with learning rate = 0.00010...
[L-1] Build 0th residual block 0 with 64 channels
[L-2] Build 0th connection layer 1 from 64 to 96 channels
[L-3] Build 1th residual block 0 with 96 channels
[L-4] Build 1th residual block 1 with 96 channels
[L-5] Build 1th residual block 2 with 96 channels
[L-6] Build 1th connection layer 3 from 96 to 128 channels
[L-7] Build 2th residual block 0 with 128 channels
[L-8] Build 2th residual block 1 with 128 channels
[L-9] Build 2th residual block 2 with 128 channels

Could you give me some suggestions?
thank you！

**Comment by panicwq** [comment]
By the way, what is your GPU configuration?
